packages/webview_flutter/webview_flutter_wkwebview/darwin/webview_flutter_wkwebview/Sources/webview_flutter_wkwebview/ScrollViewProxyAPIDelegate.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#if os(iOS)
```

Comment:
Note: These could be changed to `canImport(UIKit)` so that other platforms, like tvOS, could be used without changes. But I went with what was used in the `Objective-C` implementation.